proposals/NNNN-test-scripts-common-modules-refactoring.md

```diff
@@ -19,6 +19,8 @@
 
 All these functionality are not tests artifacts but tools that tests use.
 
+The purpose of this proposal is to clean up these common modules that everyone be able to use them while creating ATF test scripts.
```

Comment:
Please update to "...clean up these common modules that everyone is able to use while creating ATF test scripts."

Reply:
@theresalech Proposal and PR's description has been updated as you suggested. 
Thank you for the review.